### ISSUE TITLE
Not Compatible with passport 0.6.0

### ISSUE THREAD

**Issue by NoeSamaille** [ibm-cloud-security/appid-serversdk-nodejs#270]
When working with `passport@0.6.0` authentication works but the SDK is not able to retrieve access token, which prevents retrieval of user scopes and integrations eg APIs:
```
[2022-05-31T15:24:58.352] [WARN] appid-webapp-strategy - access token scope could not be found on request's session
```

It works fine with `passport@0.5.2`

**Comment by gian1200** [comment]
Also logout is now async

https://medium.com/passportjs/fixing-session-fixation-b2b68619c51d

**Comment by lucaskuperman** [comment]
Also, AppID cant redirect to original url anymore due to session regeneration starting at passport 0.6.0. AppID should catch up. More info [here](https://medium.com/passportjs/fixing-session-fixation-b2b68619c51d)

**Comment by spraju92** [comment]
Passport is coming as White Source vulnerability . We need to upgrade soon.

**Comment by dgantait** [comment]
We are also having same issue, appid is not compatible passport 0.6.0. Is there any update on this.

**Comment by vladimiratanasov** [comment]
The App ID team is currently working on a solution. Passport 0.6.0 introduces some breaking changes in their APIs that require code changes on our end. We are hoping to deliver a new version in the upcoming couple of weeks

**Comment by vladimiratanasov** [comment]
We have released SDK version 6.2.7 which uses Passport 0.6.0 under the hood
